--- configure.ac	2004-08-24 16:23:11.000000000 +0200
+++ configure.ac.oden	2004-11-10 01:20:52.953418812 +0100
@@ -47,34 +47,8 @@
 dnl look for ptlib, use a preference order of explicit PWLIBDIR, directory
 dnl at same level, home directory, /usr/local or /usr.
 
-if test "${PWLIBDIR:-unset}" != "unset" ; then
-  AC_CHECK_FILE(${PWLIBDIR}/version.h, HAS_PTLIB=1)
-fi
-if test "${HAS_PTLIB:-unset}" = "unset" ; then
-  AC_CHECK_FILE(${OPENH323DIR}/../pwlib/version.h, HAS_PTLIB=1)
-  if test "${HAS_PTLIB:-unset}" != "unset" ; then
-    PWLIBDIR="${OPENH323DIR}/../pwlib"
-  else
-    AC_CHECK_FILE(${HOME}/pwlib/include/ptlib.h, HAS_PTLIB=1)
-    if test "${HAS_PTLIB:-unset}" != "unset" ; then
-      PWLIBDIR="${HOME}/pwlib"
-    else
-      AC_CHECK_FILE(/usr/local/include/ptlib.h, HAS_PTLIB=1)
-      if test "${HAS_PTLIB:-unset}" != "unset" ; then
-        AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/local/bin)
-      else
-        AC_CHECK_FILE(/usr/include/ptlib.h, HAS_PTLIB=1)
-        if test "${HAS_PTLIB:-unset}" != "unset" ; then
-          AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/share/pwlib/make/)
-        fi
-      fi
-    fi
-  fi
-fi
-
 if test "${HAS_PTLIB:-unset}" = "unset" ; then
-  echo "Cannot find pwlib - please install or set PWLIBDIR and try again"
-  exit
+          AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/bin)
 fi
 
 if test "${PWLIBDIR:-unset}" = "unset" ; then
